homeassistant/components/met/strings.json

```diff
@@ -12,6 +12,6 @@
         }
       }
     },
-    "error": { "name_exists": "Location already exists" }
+    "error": { "name_exists": "[%key:common::config_flow::abort::already_configured_device%]" }
```

Comment:
Please change the key `name_exists` to `already_configured` too.

Reply:
In the review for NWS strings, it was suggested to use `already_configured_service`.  Which one is preferred here?

Reply:
I'd say service since it's not a device.